File: graphql-api-java/demo/src/main/java/com/graphql/demo/model/Playlist.java
package com.graphql.demo.model;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.persistence.*;
import java.util.List;

@Getter
@Setter
@NoArgsConstructor
@Entity
@Table(name = "PLAYLIST")
public class Playlist {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Long id;

    @Column(name = "NAME")
    private String name;

    @OneToMany(mappedBy = "playlist")
    private List<Song> songs;

    @ManyToOne
    @JoinColumn( name = "PERSON_ID", nullable = false)
    private Person person;

    public Playlist(String name, Person person) {
        this.name = name;
        this.person = person;
    }
}

File: graphql-api/demo/src/main/java/com/graphql/demo/model/Playlist.kt
package com.graphql.demo.model

import lombok.Getter
import lombok.NoArgsConstructor
import lombok.Setter
import javax.persistence.*

@Getter
@Setter
@NoArgsConstructor
@Entity
@Table(name = "PLAYLIST")
class Playlist(
        @field:Column(name = "NAME")
        private val name: String,

        @field:JoinColumn(name = "PERSON_ID", nullable = false)
        @field:ManyToOne
        private val person: Person
) {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    public val id: Long? = null

    @OneToMany(mappedBy = "playlist")
    private val songs: List<Song>? = null
}